packages/frontend/src/components/NavBar/GlobalSearch/index.tsx

```diff
@@ -165,38 +168,61 @@ const GlobalSearch: FC<GlobalSearchProps> = ({ projectUuid }) => {
     const { items, isSearching } = useDebouncedSearch(projectUuid, query);
 
     const searchItems = useMemo(() => {
-        return items.map<SpotlightAction>((item) => ({
-            item,
-            icon: <SearchIcon searchItem={item} />,
-            title: item.title,
-            description: item.description,
-            onTrigger: () => {
-                track({
-                    name: EventName.SEARCH_RESULT_CLICKED,
-                    properties: {
-                        type: item.type,
-                        id: getSearchResultId(item.item),
-                    },
-                });
-                track({
-                    name: EventName.GLOBAL_SEARCH_CLOSED,
-                    properties: {
-                        action: 'result_click',
-                    },
-                });
+        return items.map<SpotlightAction>((item) => {
+            const isSearchItemWithValidationError =
+                item.item &&
+                ['dashboard', 'saved_chart'].includes(item.type) &&
+                'validationErrors' in item.item &&
+                item.item.validationErrors?.length > 0;
```

Comment:
this can be a `itemHasValidationError(item: ItemType)` and can be extracted outside of the component completely

Reply:
that's great, will change that!